.idea/misc.xml

```diff
@@ -35,5 +38,5 @@
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="false" project-jdk-name="1.8" project-jdk-type="JavaSDK" />
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK" />
 </project>
```

Comment:
Maybe we should not bother ourselves with EOLs in IDEA configs? The IDE is always removing it.

Reply:
What if we start using pre-commit hooks that do this, instead? Or, Gradle plugins similarly to how JUnit does it?